background/lookup.js

```diff
@@ -56,7 +58,9 @@ export default class Lookup {
 
   _reverseEmail (input) {
     if (!this._emails[input]) {
-      this._emails[input] = fetch(`https://id.parity.io:8443/?email=${input}`)
+      const hash = keccak_256(input);
+
+      this._emails[input] = fetch(`https://id.parity.io:8443/?emailHash=0x${hash}`)
```

Comment:
So I think this would eventually be an option of the Extension, right @tomusdrw ? I've put the testnet there for testing, but I think there should be another PR for more options (port, etc...)